docs/release-notes/2.31.0.md

```diff
@@ -0,0 +1,159 @@
+
+- PMM now supports Percona Operator for MongoDB [1.12.0](https://docs.percona.com/percona-operator-for-mongodb/RN/Kubernetes-Operator-for-PSMONGODB-RN1.12.0.html).
+
+
+
+
+
+
+
+## New Features
+
+- [PMM-10089](https://jira.percona.com/browse/PMM-10089): We have upgraded Grafana to 9.1 in PMM 2.31.0.
+
+- [PMM-10092](https://jira.percona.com/browse/PMM-10092): We have expanded and improved the main menu for PMM 2.31.0 based on Grafana 9.1. The main menu contains crucial elements, one of them being starred dashboards. 
+
+- [PMM-10467](https://jira.percona.com/browse/PMM-10467):  
+
+- [PMM-10443](https://jira.percona.com/browse/PMM-10443): 
+
+- [PMM-10244](https://jira.percona.com/browse/PMM-10244): 
+
+
+ 
+## Improvements
+
+- [PMM-10560](https://jira.percona.com/browse/PMM-10340): We have improved the MongoDB Collection Details Dashboard. The Collection Activity panel is now collapsed by default, and the graphs are visible only when the`--enable-all-collectors` parameter is passed in pmm-admin command.
+
+  
+- [PMM-10330](https://jira.percona.com/browse/PMM-10330): Simpler DBaaS and Percona Platform connection configuration by suggesting the public address of the PMM server.
+
+- [PMM-10495](https://jira.percona.com/browse/PMM-10495): By default, Swagger UI attempts to validate specs against swagger.io’s online validator. For PMM 2.31.0, we have disabled it to safeguard your privacy.
+
+
+- [PMM-10064](https://jira.percona.com/browse/PMM-10064): Introduced UI changes on the PMM Inventory pages for better readability and user experience.
+
+
+
+- [PMM-10076](https://jira.percona.com/browse/PMM-10076): For an improved UI experience, we have enforced the enter button for the date picker, which performs the same action as clicking `apply time range`.
+
+- [PMM-7806](https://jira.percona.com/browse/PMM-7806): Upgraded postgres_exporter version used in pmm from 0.8.0 to 0.10.1. With this upgrade, you can access the latest features with all the bug fixes in place. 
+
+  
+- [PMM-10018](https://jira.percona.com/browse/PMM-10018): Lowered the default values of CPU and memory for haproxy while deploying a DBaaS [Percona XtraDB Cluster](https://www.percona.com/software/mysql-database/percona-xtradb-cluster) (PXC) cluster to avoid wasting resources.
+
+
+    
+- [PMM-9118](https://jira.percona.com/browse/PMM-9118): Refined the error message thrown when a secure connection cannot be established while adding a monitoring database to PMM, thus making it much more contextual and easy to troubleshoot.
+
+
+- [PMM-10516](https://jira.percona.com/browse/PMM-10516): 
+
+- [PMM-10095](https://jira.percona.com/browse/PMM-10095): While removing the service with pmm-admin, now you do not have to specify the service name for a single service. PMM will automatically look up this service and remove it.
+
+- [PMM-9989](https://jira.percona.com/browse/PMM-9989): PMM now supports Percona Operator for MongoDB 1.12.0.
```

Comment:
no, we don't. CC @gen1us2k @puneet0191 ticket is still In Progress

Reply:
The issue is still in progress. We do not support the 1.12 PSMDB operator yet.

Reply:
Will remove it..